### DISCUSSION TITLE
Руководство ПЛИ, тело документа, 1.3 Секция "Информация об использованном оборудовании и расходных материалах" ANALYSERS

### DISCUSSION THREAD

**Discussion by eugenekogan-spb** [fhir-ru/core#282] [Лабораторная РГ]
Секция предназначена для указания информации о лабораторном оборудовании и расходных материалах, непосредственно использованных для проведения лабораторных исследований.
Идентификатор, присвоенный лабораторному оборудованию или расходному материалу в данной секции, будет использоваться как ссылка, направляющая на его описание в данной секции из кодированных элементов в секции "Результаты лабораторных исследований" ("Результаты проведенных исследований").
В данную секцию включается следующая информация:
• Идентификатор из справочника оборудования ЛИС (при наличии информации);
• Наименование из справочника оборудования ЛИС.

**Comment by eugenekogan-spb** [comment]
<img width="767" alt="image" src="https://github.com/user-attachments/assets/878ab447-99a6-4543-ab24-57ee0d547ad2">
<img width="767" alt="image" src="https://github.com/user-attachments/assets/44a78307-7304-44d7-8a70-727e4e91d637">

**Reply by eugenekogan-spb** [reply]
Оборудование передается в ресурсе Device

Instance: device-sysmex
InstanceOf: Device
Usage: #example

* identifier
  * system = "1.2.643.5.1.13.13.12.2.22.1576.100.1.1.67"
  * value = "3455"
  * type
    * text = "Идентификатор анализатора из справочника оборудования МИС"

* deviceName
  * name = "Гематологический анализатор Sysmex KX21"
  * type = #user-friendly-name

// для R5 * displayName = "Гематологический анализатор Sysmex KX21"

**Reply by eugenekogan-spb** [reply]
Ссылка на device будет указана в тех Observation, которые измерены с его помощью.

**Comment by eugenekogan-spb** [comment]
У3-2:Секция "Информация об использованном оборудовании и расходных материалах" ОБЯЗАНА иметь один [1..1] элемент entry, содержащий формализованное перечисление использованного оборудования и расходных материалов, который:
1. ОБЯЗАН иметь один [1..1] элемент organizer (атрибут @classCode должен иметь значение "CLUSTER", атрибут @moodCode должен иметь значение "EVN"), который:
1.1. ОБЯЗАН иметь один [1..1] элемент statusCode (атрибут @code должен иметь значение "completed");
1.2. ОБЯЗАН иметь хотя бы один [1..*] элемент participant (атрибут @typeCode должен быть представлен с не пустым значением), содержащий сведения об использованном оборудовании или расходных материалах. ЕСЛИ кодируется оборудование, ТО атрибут participant/@typeCode должен иметь значение "DEV"; ЕСЛИ кодируются расходные материалы, ТО атрибут participant/@typeCode должен иметь значение "CSM". Данный элемент:
1.2.1. ДОЛЖЕН иметь один [1..1] элемент participantRole (атрибут @classCode должен иметь значение "ROL"), который:
1.2.1.1. ДОЛЖЕН иметь один [1..1] элемент id (атрибуты @root, @extension должны быть представлены с не пустыми значениями; атрибут @root должен быть сформирован по правилу формирования идентификаторов оборудования и расходных материалов для invitro диагностики, т.е. "OID_медицинской_организации.100.НомерМИС.НомерЭкзМИС.67", атрибут @extension должен соответствовать идентификатору оборудования и расходных материалов для invitro диагностики), содержащий идентификатор из справочника оборудования ЛИС;
1.2.1.2. ОБЯЗАН иметь один [1..1] элемент playingDevice, который:
1.2.1.2.1. ОБЯЗАН иметь один [1..1] элемент manufacturerModelName, содержащий наименование из справочника оборудования ЛИС, который должен иметь не пустое наполнение.

**Comment by maximalexeev** [comment]
По поводу передачи использованных расходных материалов - предложил бы передавать их как Observation.method (Методика исследования и/или использованные материалы)
по справочникам 
https://nsi.rosminzdrav.ru/dictionaries/1.2.643.5.1.13.13.99.2.422/passport/2.1
https://nsi.rosminzdrav.ru/dictionaries/1.2.643.5.1.13.13.99.2.660/passport/2.10
(или пользовательский справочник, т.к. более подходящих пока, к сожалению, пока нет)

**Reply by eugenekogan-spb** [reply]
Это расходные материалы для ВМП и тест-системы для Ковид-19. Вряд ли подойдет.
В спецификации СЭМД написано "по справочнику лаборатории". Надо так и сделать. 
Требуется изобрести system для этого кода, напримерпо образцу и подобию OID для других кодов, присвоенных МО.
Предлагается передавать в расширении.

**Reply by eugenekogan-spb** [reply]
System можно не изобретать, он предложен в спецификации СЭМД - 1.2.643.5.1.13.13.12.2.22.1576.100.1.1.67.
Считается, что это "справочник оборудования и расходных материалов лаборатории".
